website/docs/reference/warehouse-setups/snowflake-setup.md

```diff
@@ -212,6 +216,7 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
 | retry_on_database_errors | No | A boolean flag indicating whether to retry after encountering errors of type [snowflake.connector.errors.DatabaseError](https://github.com/snowflakedb/snowflake-connector-python/blob/ffdd6b3339aa71885878d047141fe9a77c4a4ae3/src/snowflake/connector/errors.py#L361-L364) |
 | connect_retries | No | The number of times to retry after an unsuccessful connection |
 | connect_timeout | No | The number of seconds to sleep between failed connection retries |
+| release_connection | No | Drop and reestablish connections to Snowflake for each. |
```

Comment:
Worth adding the default setting here so users can know where to opt-in or opt-out

Reply:
Agreed, and Jerco suggested because we should have things default to False I'll have to rename the param. Will do that before flipping this PR from draft to ready for review PR.